app/models/calendar_event.rb

```diff
@@ -119,7 +119,7 @@ def initialize_loan_start(loan)
 
   def initialize_loan_end(loan)
     @start = loan.target_end_date
-    @title = "End " + loan.name
+    @title = "End " + loan.display_name
```

Comment:
Should be using string interpolation here and above. Please try to avoid `+` with strings. If you can't use interpolation you should be using `<<`. Please look up the difference and let me know if it's not clear.